kanalony-storage-access/src/main/scala/kanalony/storage/generator/CassandraCqlGenerator.scala

```diff
@@ -0,0 +1,47 @@
+package kanalony.storage.generator
+
+import java.io.File
+
+import kanalony.storage.utils.fs
+
+/**
+ * Created by elad.benedict on 3/27/2016.
+ */
+object CassandraCqlGenerator {
+
+  def main (args: Array[String]) {
+
+    val keySpaceName = "kanalony_user_activity"
+    var res = List(s"CREATE KEYSPACE IF NOT EXISTS ${keySpaceName} WITH replication = {'class': 'SimpleStrategy', 'replication_factor': '1'}  AND durable_writes = true;")
```

Comment:
Where would you like it? I can put it in a separate file. Currently (prior to this commit) it's part of the table creation script